app/assets/stylesheets/movies.scss

```diff
@@ -395,6 +397,12 @@ article.tile-cover-pic {
     } /* .col-xs-9 */
   } /* .modal */
 
+  // article.tile-cover-pic {
+  //   #modal-trigger {
+  //     display: none;
+  //   }
+  // }
+
```

Comment:
Did you mean to leave this here? If so, that's fine; I'm just checking.

Reply:
Nooooope! I'll cut that back out.